README.md

```diff
@@ -2,85 +2,36 @@
 
 This repository contains the Apache DataFusion blog at https://datafusion.apache.org/blog/
 
-## Setup for Mac
```

Comment:
I don't fully understand how this all works, but it might be nice to list out the steps required to make a new blog post (will making a PR and merging it publish the content 🤔  -- will it also generate a preview?)